manifests/release/demo-content.yaml.in

```diff
@@ -0,0 +1,34 @@
+# Virtual Machine Presets
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachinePreset
+metadata:
+  name: windows-server-2012r2
+  selector:
+    matchLabels:
+      kubevirt.io/os: win2k12r2
+spec:
+  terminationGracePeriodSeconds: 0
```

Comment:
The terminationGracePeriod should not be in the preset.

Reply:
@rmohr  done